### PR TITLE
v0.7.0

### PR DESCRIPTION
**Pull request by cmatheson**
* Updates user information when refreshing
* Bumps WorkOS client to 7.12.0

### CODE REVIEW

File: src/workos.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
 import { WorkOS } from '@workos-inc/node';
 import { WORKOS_API_HOSTNAME, WORKOS_API_HTTPS, WORKOS_API_KEY, WORKOS_API_PORT } from './env-variables.js';
+import { version } from '../package.json';
 
 
 const options = {
   apiHostname: WORKOS_API_HOSTNAME,
   https: WORKOS_API_HTTPS ? WORKOS_API_HTTPS === 'true' : true,
   port: WORKOS_API_PORT ? parseInt(WORKOS_API_PORT) : undefined,
+  appInfo: {
```
Comment (cmatheson) [inline src/workos.ts:10]: this is new in WorkOS v7

File: tsconfig-base.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "jsx": "react",
     "module": "ES2020",
     "moduleResolution": "Node",
+    "resolveJsonModule": true,
```
Comment (mthadley) [inline tsconfig-base.json:8]: Nice. Any reason not to turn this on and do the same in `@workos-inc/node` for the [`VERSION` constant](https://github.com/workos/workos-node/blob/80edc61076eb9a2749756051a083a947198497cf/src/workos.ts#L34)?

**Review by mthadley — APPROVED** [review]

File: src/workos.ts [hunk 3]
```diff
@@ -1,11 +1,16 @@
 import { WorkOS } from '@workos-inc/node';
 import { WORKOS_API_HOSTNAME, WORKOS_API_HTTPS, WORKOS_API_KEY, WORKOS_API_PORT } from './env-variables.js';
+import { version } from '../package.json';
```
Comment (PaulAsjes) [inline src/workos.ts:3]: I'm fairly certain this will break in production, as our tsconfig doesn't know how to handle json files.

**Review by PaulAsjes — CHANGES_REQUESTED** [review]

File: src/workos.ts (hunk 3 above)
Comment (PaulAsjes) [reply src/workos.ts:3]: I'm dumb and didn't see that you added `resolveJsonModule`.

**Review by PaulAsjes — APPROVED** [review]

File: src/workos.ts (hunk 3 above)
Comment (cmatheson) [reply src/workos.ts:3]: well, the json part is fine but it actually does break production 😂 😭 . it changes the compiled output such that all the ts files get nested under `src` and breaks things. going to push up a new iteration.

**Pull request merged**